packages/kit/src/core/generate_manifest/find_server_assets.js

```diff
@@ -0,0 +1,53 @@
+import { find_deps } from '../../exports/vite/build/utils.js';
+
+/**
+ * Generates the data used to write the server-side manifest.js file. This data is used in the Vite
```

Comment:
"generates" makes me think it's doing more than creating an in-memory representation
```suggestion
 * Returns the data used to write the server-side manifest.js file. This data is used in the Vite
```

Reply:
I think I might have just copy-pasted this from elsewhere, the description is totally wrong